src/api/config/initializers/marcel.rb

```diff
@@ -0,0 +1,11 @@
+require 'marcel'
+require 'marcel/mime_type'
+
+# Marcel uses Apache Tika mime database, which is less complete than Freedesktop one
+# Marcel::Magic is based on MimeMagic which uses the Freedesktop mime database, so it's
+# worth considering using it in order to avoid the below configuration
```

Comment:
Not sure what you want to say to me with this comment? That we shouldn't do this but something else? :)

Reply:
I would rather have you explain why you are going down this route and not the other in the commit message :)